source/module_basis/module_nao/pswfc_radials.cpp

```diff
@@ -0,0 +1,308 @@
+#include "module_basis/module_nao/pswfc_radials.h"
+#include "module_base/tool_quit.h"
+#include "module_base/math_integral.h"
+#include <algorithm>
+#include <cmath>
+
+PswfcRadials& PswfcRadials::operator=(const PswfcRadials& rhs)
+{
+    RadialSet::operator=(rhs);
+    return *this;
+}
+
+void PswfcRadials::build(const std::string& file, 
```

Comment:
for pswfc, it is the same as hydrogen, build is the pure virtual, central function